src/API/Common/Export/Headers.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Common\Export;
+
+class Headers
+{
+    const EXPORTER_HEADER = 'X-Is-Otel-Exporter';
```

Comment:
I chose a new class in `API` rather than adding this to `PsrTransport` so that auto-instrumentations did not need to depend on `SDK`...